pkg/kubelet/cm/dra/state/state_checkpoint_test.go

```diff
@@ -59,17 +59,14 @@ func TestCheckpointGetOrCreate(t *testing.T) {
 			expectedState:     []ClaimInfoState{},
 		},
 		{
-			description:       "Restore checkpoint - single claim",
-			checkpointContent: `{"version":"v1","entries":[{"ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"DriverState":{"test-driver.cdi.k8s.io":{"Devices":[{"PoolName":"worker-1","DeviceName":"dev-1","RequestNames":["test request"],"CDIDeviceIDs":["example.com/example=cdi-example"]}]}}}],"checksum":1925941526}`,
+			description:       "Restore-checkpoint-V1-single-claim",
+			checkpointContent: `{"version":"v1","entries":[{"DriverName":"test-driver.cdi.k8s.io","ClassName":"class-name","ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"ResourceHandles":[{"driverName":"test-driver.cdi.k8s.io","data":"{\"a\": \"b\"}"}],"CDIDevices":{"test-driver.cdi.k8s.io":["example.com/example=cdi-example"]}}],"checksum":1812793841}`,
 			expectedState: []ClaimInfoState{
```

Comment:
TODO: these test cases should use [checkpont content from k/k 1.30](https://github.com/kubernetes/kubernetes/blob/release-1.30/pkg/kubelet/cm/dra/state/state_checkpoint_test.go#L62).

Reply:
done